esi_leap/common/statuses.py

```diff
@@ -20,4 +20,7 @@
 WAIT_EXPIRE = 'wait expire'
 WAIT_FULFILL = 'wait fulfill'
 
-CANCELLABLE_LEASE_STATUSES = [ACTIVE, CREATED, WAIT_FULFILL]
+OFFER_CAN_DELETE = (AVAILABLE, ERROR)
+LEASE_CAN_DELETE = [ACTIVE, CREATED, ERROR, WAIT_FULFILL]
+
+#    CANCELLABLE_LEASE_STATUSES = [ACTIVE, CREATED, WAIT_FULFILL]
```

Comment:
No need to comment this out; you can just remove it!

Reply:
okay!